Gemfile

```diff
@@ -40,18 +40,18 @@ gem 'rubytree'
 gem 'statsd-ruby'
 gem 'uglifier', '>= 1.3.0'
 
-gem 'advice_plans', '~> 3.1.0'
+gem 'advice_plans', git: 'git@github.com:moneyadviceservice/advice_plans.git', branch: 'unpin_sass_rails'
```

Comment:
Yeah, I was thinking that when I looked through it... no idea why I didn't. Anyway, luckily this bit won't end up getting merged.